yeat/workflows/snakefiles/Paired

```diff
@@ -146,7 +146,7 @@ rule spades:
     shell:
         """
         spades.py -1 {input.read1} -2 {input.read2} -t {threads} -o {params.outdir} {params.extra_args}
-        ln -s scaffolds.fasta {output.contigs}
+        ln -s contigs.fasta {output.contigs}
```

Comment:
Talked with the analysts and they said that don't use the `scaffolds.fasta`. Instead, they use `contigs.fasta`. In the past, `runMA` used `scaffolds` over the base contigs file. The reason for one file over the other is unknown.

If the user needs to look at the `scaffolds.fasta`, they can access it in the spades output directory.

According to the SPAdes user manual:
```
<output_dir>/scaffolds.fasta contains resulting scaffolds (recommended for use as resulting sequences)
<output_dir>/contigs.fasta contains resulting contigs
```